extensions/sql-migration/src/api/azure.ts

```diff
@@ -62,6 +63,30 @@ export async function getLocations(account: azdata.Account, subscription: Subscr
 	return filteredLocations;
 }
 
+export async function getArcLocations(account: azdata.Account, subscription: Subscription): Promise<azurecore.azureResource.AzureLocation[]> {
```

Comment:
Doesnot this method should be named as getSqlServerAzurearcLocation or getAzureArcLocation